nls/kappnav.properties

```diff
@@ -303,3 +303,4 @@ edit.207 = Unable to edit the resource because of a Kubernetes exception. The ex
 
 run.audit=Run Audit
 run.audit.action.description = Run Inventory Command Action
+table.empty.command.actions = No actions have been submitted yet. Want to 
```

Comment:
I'm sure the rest of this is some kind of link, however, it may not translate correctly.

Reply:
Yeah, I was thinking about this, too. This is the wording that was provided by the design team. I'll play with the wording.